frontend/src/employee-frontend/components/payments/Actions.tsx

```diff
@@ -46,6 +60,19 @@ const Actions = React.memo(function Actions({
           <Gap size="s" horizontal />
         </>
       ) : null}
+      <AsyncButton
+        text={i18n.payments.buttons.deletePayment(checkedIds.length)}
+        disabled={checkedIds.length === 0}
+        onClick={() => deletePayments(checkedIds)}
+        onSuccess={() => {
+          setError(undefined)
+          actions.clearChecked()
+          reloadPayments()
+        }}
+        onFailure={() => setError(i18n.common.error.unknown)}
```

Comment:
Showing a generic error message to the user is not needed IMO. There will be a red cross ❌ indicating an error in the button, which gives the same amount of information to the user as the generic error.